mobile/src/bundles/common/enums/app/app-environment-mobile.enum.ts

```diff
@@ -0,0 +1,7 @@
+const AppEnvironmentMobile = {
```

Comment:
```suggestion
const OnboardingStepState = {
```
I made comment with an error

Reply:
And I' ve created this enum in the right folder?